index.html

```diff
@@ -5,7 +5,7 @@
 <div class="home">
   <section>
     <h2 class="smallcap">about</h2>
-      <p>This is the personal website of me, Steven Angles. Here you will find various musings on software engineering, technology and more. I write code for a living and also do it in my free time. If I'm not writing code, then you can find me at various festivals and concerts around Columbus, wrenching on a car or reading up on aviation.</p>
+      <p>This is the personal website of me, Steven Angles. Here you will find various musings on software engineering, technology, and more. I write code for a living and also do it in my free time. If I'm not writing code, then you can find me at various festivals and concerts around Columbus, wrenching on a car, or reading up on aviation.</p>
```

Comment:
see, I _was_ going to approve this...but then you had to go and be unreasonable.